### PR TITLE
Log runtime so parallel_tests can equalize test runtime.

### PR DESCRIPTION
**Pull request by fonglh**

### CODE REVIEW

**Comment by fonglh** [comment]
@allenwq ready for review and merge.

**Comment by allenwq** [comment]
@fonglh Runtime will be updated every time ?

**Comment by fonglh** [comment]
I think so, it doesn't seem all that successful though. I followed the steps from https://github.com/grosser/parallel_tests#even-test-group-run-times

File: .travis.yml [hunk 1]
```diff
@@ -45,6 +48,9 @@ before_script:
   - psql -c 'create database coursemology_test2;' -U postgres
 #  - bundle exec rake db:setup
   - bundle exec rake parallel:setup[2]
+  # Copy runtime log if it exists. Create runtime cache directory if absent.
+  - 'if [ -f "$TRAVIS_BUILD_DIR/spec-runtime/parallel_runtime_rspec.log" ]; then cp $TRAVIS_BUILD_DIR/spec-runtime/parallel_runtime_rspec.log $TRAVIS_BUILD_DIR/tmp/parallel_runtime_rspec.log; cat $TRAVIS_BUILD_DIR/tmp/parallel_runtime_rspec.log; fi'
```
Comment (allenwq) [inline .travis.yml]: Should this be put before `bundle exec rake parallel:setup[2]`, Or I understand wrongly ?

Comment (fonglh) [inline .travis.yml]: I think it doesn't matter. `setup` just initializes the databases. The log should only be used for actually running the tests.

Comment (allenwq) [inline .travis.yml]: `bundle exec rake parallel:setup[2]` is running the actual spec.  The log should be copied over before that ?

Comment (allenwq) [inline .travis.yml]: Oh, looked wrongly.

**Comment by allenwq** [comment]
Doesn't seem to be balanced:
Finished in 8 minutes 39 seconds (files took 12.47 seconds to load)
Finished in 10 minutes 11 seconds (files took 7.46 seconds to load)

But let's merge and keep monitoring first.

**Pull request merged**